tests/utils.py

```diff
@@ -0,0 +1,60 @@
+from eth_account.signers.local import LocalAccount
+
+from gnosis.eth.contracts import get_multi_send_contract
+from gnosis.eth.tests.utils import deploy_erc20
+
+
+def send_tx(w3, tx, account: LocalAccount):
```

Comment:
This function is already defined on `gnosis.eth.tests.utils`. Please don't copy functions that they are already defined (unless you need to change something 😉 )